packages/nuxt/src/client/piniaIntegration.ts

```diff
@@ -0,0 +1,39 @@
+import { defineIntegration } from '@sentry/core';
+import type { IntegrationFn } from '@sentry/types';
+
+import { consoleSandbox } from '@sentry/utils';
+import { createSentryPiniaPlugin } from '@sentry/vue';
+
+const INTEGRATION_NAME = 'Pinia';
+
+type Pinia = { use: (plugin: ReturnType<typeof createSentryPiniaPlugin>) => void };
+
+const _piniaIntegration = ((
+  // `unknown` here as well because usePinia declares this type: `export declare const usePinia: () => unknown;`
+  pinia: unknown | Pinia,
+  options: Parameters<typeof createSentryPiniaPlugin>[0] = {},
+) => {
+  return {
+    name: INTEGRATION_NAME,
+    setup() {
+      if (!pinia || (typeof pinia === 'object' && !('use' in pinia))) {
+        consoleSandbox(() => {
+          // eslint-disable-next-line no-console
+          console.warn(
+            '[Sentry] You added the Pinia integration, but the passed parameter `pinia` has the wrong value. Make sure to enable Pinia by adding `"@pinia/nuxt"` to your Nuxt modules array and pass pinia to Sentry with `piniaIntegration(usePinia())`. Current value of `pinia`: ',
```

Comment:
l: You don't need the extra space after `:`, a space is rendered by default between arguments passed to console logs.